### PR TITLE
fix nanomsg tag

### PR DESCRIPTION
**Pull request by RalfKliemt**

### CODE REVIEW

**Comment by RalfKliemt** [comment]
Also Genfit and Eigen (dependency of GenFit) are added.
I'm not sure where I got them from, but I had those lying around.

Cheers!

File: eigen.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package: Eigen
+version:  "3.3.4"
+build_requires:
+  - CMake
+---
+#/bin/sh
```
Comment (dennisklein) [inline eigen.sh]: Use `#!/bin/sh -e`

File: genfit.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+version:  "%(tag_basename)s"
+source: https://github.com/GenFit/GenFit
+tag: master
+requires:
+  - ROOT
+  - Eigen
+build_requires:
+  - CMake
+---
+#!/bin.sh
```
Comment (dennisklein) [inline genfit.sh]: Use `#!/bin/sh -e`

File: eigen.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  curl -Lo $file "-z '$file'" "$URL"
+else
+  curl -Lo $file "$URL"
+  #md5sum $file > $file".md5"
+fi
+# extract, but do'nt overwrite existing files
+#if md5sum $file -c $file".md5"
+#then
+tar -xjf $file -C $SOURCEDIR --strip-components=1
+#fi
```
Comment (dennisklein) [inline eigen.sh]: Lines 7-23 are working around aliBuild, you need to specify a git repo instead. In the README file in the alfadist repo you find a section on what to do when there is no git repo yet. Pls reference a git repo and remove lines 7-23. Thx!

File: genfit.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+printenv
+echo "ROOT_INCLUDE_PATH = "$ROOT_INCLUDE_PATH
+echo "EIGEN_INCLUDE_PATH = "$EIGEN_INCLUDE_PATH
+cmake $SOURCEDIR \
+  -DCMAKE_INSTALL_PREFIX:PATH="$INSTALLROOT"
+make ${JOBS+-j $JOBS}
+make install
+#ROOT_ROOT
```
Comment (dennisklein) [inline genfit.sh]: Remove line 19

File: eigen.sh [hunk 5]
```diff
@@ -0,0 +1,45 @@
+package: Eigen
```
Comment (dennisklein) [inline eigen.sh]: Make sure, you match the name of the git repo exactly.

**Review by dennisklein — CHANGES_REQUESTED** [review]
Thx for the fixes and new recipes. I commented a couple of small things. Downloading the source in the build script should be avoided. AliBuild currently supports git only, find some hints on what to do in cases there is no git repo in the README file here. Thx!

**Comment by RalfKliemt** [comment]
Hi Dennis,
I implemented the requested changes. 
Cheers!
  Ralf

**Review by dennisklein — APPROVED** [review]

**Comment by dennisklein** [comment]
Contains fix for #10

**Review by fuhlig1 — APPROVED** [review]

**Pull request merged**